stream/stream_test.go

```diff
@@ -326,21 +327,23 @@ func (s *STSuite) TestPreservesTLS(c *C) {
 }
 
 func BenchmarkLoggingDebugLevel(b *testing.B) {
-	streamer, _ := New(noOpNextHttpHandler{})
+	l := log.StandardLogger()
+	streamer, _ := New(noOpNextHttpHandler{}, Logger(l))
```

Comment:
I think the changes in this benchmark are unnecessary, as the default is logrus' StandardLogger, and `SetLevel` und `SetOutput` work on that logger.